README.md

```diff
@@ -1 +1,24 @@
-# TODO
+# Fixie Javascript SDK
+
+This is a monorepo providing a TypeScript / JavaScript SDK to the
+[Fixie](https://fixie.ai) platform. It contains the following packages:
+
+- `fixie`: A NodeJS SDK and CLI.
+- `fixie-web`: A browser-based SDK.
+- `fixie-common`: A shared package containing code used by both the
+  NodeJS and browser SDKs.
+
+Full documentation is provided at [https://fixie.ai/docs](https://fixie.ai/docs).
```

Comment:
could we highlight the differences between web & common here? voice is the only thing right now?

Reply:
I don't think we want to stuff a lot of details into this README right now, but if you want to flesh this out as part of the documentation refactor please go for it. My concern is generally around duplicating information in multiple places and having to keep it up to date across the README and the "real" documentation. This PR is not really about the documentation, it's for the code structuring.